packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy.scss

```diff
@@ -1,36 +1,29 @@
-.dc-modal {
-    $strategy-modal: 'modal--strategy';
-
-    &__container {
-        &_#{$strategy-modal} {
-            max-height: 57.1rem !important;
+.quick-strategy {
+    $quick-strategy: &;
 
-            .modal__scrollbar {
-                max-height: 516px;
-                margin-bottom: 75px;
-            }
-            .modal__content {
-                padding: 1.6rem 2.4rem;
-                height: calc(100% - 200px);
-                max-height: 636px;
-            }
-        }
+    &__notifications-container--open {
+        display: flex;
+        justify-content: center;
     }
-}
 
-.dc-mobile-full-page-modal.quick-strategy__wrapper {
-    height: 100%;
-    overflow-y: visible;
-}
+    &__container {
+        width: 77.5rem;
+        transform: translateY(0);
+    }
 
-.quick-strategy {
-    $quick-strategy: &;
+    &__title {
```

Comment:
we can use Text as a wrapper for title so we can pass props to component directly

Reply:
done ✅